mysql-test/main/max_used_connections_time.test

```diff
@@ -0,0 +1,35 @@
+-- source include/not_embedded.inc
+
+--echo #
+--echo # MDEV-30543 New status variable: Max_used_connections_time
+--echo #
+
+FLUSH STATUS;
+--disable_query_log
+--disable_result_log
+connect (con1,localhost,root,,);
+--sleep 1
+connect (con2,localhost,root,,);
+--sleep 1
+let $time_1=`SELECT VARIABLE_VALUE FROM INFORMATION_SCHEMA.SESSION_STATUS WHERE VARIABLE_NAME LIKE 'max_used_connections_time'`;
+--sleep 1
+connection con1;
+disconnect con2;
+--sleep 1
+let $time_2=`SELECT VARIABLE_VALUE FROM INFORMATION_SCHEMA.SESSION_STATUS WHERE VARIABLE_NAME LIKE 'max_used_connections_time'`;
+--echo # This should report 0 as it must have the same value as above
+let $dif_1= `SELECT '$time_1' <> '$time_2'`;
+--echo $dif_1
+--sleep 1
+connect (con3,localhost,root,,);
+--sleep 1
+connect (con4,localhost,root,,);
+--sleep 1
+let $time_3=`SELECT VARIABLE_VALUE FROM INFORMATION_SCHEMA.SESSION_STATUS WHERE VARIABLE_NAME LIKE 'max_used_connections_time'`;
+--echo # This should report 1 as connection number was increased
+let $dif_2= `SELECT '$time_1' <> '$time_3'`;
+--echo $dif_2
+
+disconnect con4;
+disconnect con3;
+disconnect con1;
```

Comment:
Adding the last empty line (this also should be added in Coding standard) @LinuxJedi .

Reply:
AFAIK this isn't a standard.

Reply:
I'll put in a plug here for using [wtf](https://github.com/dlenski/wtf), which will automatically clean up whitespace inconsistencies including this one: in its default mode it will ensure that the final line ends with `\n`.

Reply:
final line ending with `\n` makes sense, more than an empty blank one anyway.